### ISSUE TITLE
img/*.ps are missing

### ISSUE THREAD

**Issue by alexander-naumov** [bluhm/talk-obsdsecurity#1]
Hello Alexander,

thanks a lot for sharing, but it seems that images (img/*.ps) are missing...
I would like to build it or download it from openbsd.org/events.html page, but there is also no link to this talk.

Cheers,
Alex

**Comment by bluhm** [comment]
On Wed, Nov 13, 2019 at 02:06:19PM -0800, anaumov wrote:
> thanks a lot for sharing, but it seems that images (img/*.ps) are missing...
> I would like to build it or download it from openbsd.org/events.html
> page, but there is also no link to this talk.

I remember there was a license issue.  A coworker of mine created
these images for his talk.  I asked him if I could use it and copied
them over locally.  So I have permission to use them, but not to
publish under an ISC license.  That is why they are not part of the
git repository.  Although it would be no problem to get them with
a proper license, I avoided that work.

I only gave this talk in a company context, so there is nothing on
openbsd.org.

bluhm
